jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/AbstractQueryRewriter.java

```diff
@@ -460,4 +460,25 @@ public Object getResultSetValue(ResultSet resultSet, int columnIndex, Column col
         }
         return resultSet.getObject(columnIndex);
     }
+
+    protected boolean isSupportedVersion(String databaseProductName, int databaseVersion) {
+
+        if(databaseProductName.equals(_dataContext.getDatabaseProductName())
+                && databaseVersion <= databaseVersionToInt(_dataContext.getDatabaseVersion())) {
+            return true;
+        }
+        return false;
+    }
+
+    private int databaseVersionToInt(String version) {
```

Comment:
I'd call this method something like `getDatabaseMajorVersion()` just because it will return the _major_ version element only, not something that represents the 'real' version.